gcc/testsuite/gcc.dg/ftz-math.c

```diff
@@ -1,9 +1,14 @@
 /* Tests -fftz-math flag */
 /* { dg-do run { target x86_64-*-* } } */
-/* { dg-options "-O2 -fftz-math" } */
+/* { dg-options "-O2 -Wall -Wpedantic -fftz-math" } */
```

Comment:
Is it a convention to have the warnings turned on in the test cases? It might not be wise as in the future gcc versions they might add warnings (that might be harmless regarding the tested features).